fendermint/vm/interpreter/src/fvm/state/exec.rs

```diff
@@ -48,6 +48,22 @@ pub struct FvmStateParams {
     pub power_scale: PowerScale,
 }
 
+/// Parts of the state which can be updated by message execution, apart from the actor state.
+///
+/// This is just a technical thing to help us not forget about saving something.
+///
+/// TODO: `base_fee` should surely be here.
```

Comment:
Nice one, I like this approach! Yes, I was actually going to ask you why the `base_fee` wasn't included here. 

I would include it for now even if it doesn't currently change as we may want to expose a callback to update it for users in the future. But maybe you have a reason not to include it yet, so happy either way.

Reply:
I didn't include it purely because there is no user story for how to update it. I created an issue, but haven't checked what's happening in Lotus or Forest. Maybe it's based on the last N blocks or something? Or maybe it's based on some voting, I don't know. I'd rather have a TODO stating it's not done, then code which suggests it might be done, which sends people looking all over to code just to confirm that nothing is happening. 

But the history of it is already maintained, so as soon as we add it here, the compiler will tell us to update the value in `app.rs` and all will be well.

Reply:
I would keep a static `base_fee` and not use what Lotus is doing, I would rather have a way for users to decide the `base_fee` function instead of giving them a really complicated one that they don't understand. So yes, let's leave it for the future. Thanks!